### ISSUE TITLE
"&nbsp;" issue

### ISSUE THREAD

**Issue by jforaker** [wrakky/react-html-parser#12]
non breaking spaces are rendered as a string `&nbsp;`

for example, paste this block into the demo site:

```
<p><span>He had a stark message for Republicans when they met today about his bill to repeal Obamacare: Vote for the bill, or you could&nbsp;</span><a class="markup--anchor markup--li-anchor" href="#">lose re-election</a><span>.</span></p>
```

You will see the `&nbsp;`'s actually get rendered. Pasting the same block into Codepen/JSBin renders the space properly as expected.

Expected: `&nbsp;` should render a space

**Comment by real34** [comment]
Same thing for other html special chars (`&eacute;` ...)

**Comment by jforaker** [comment]
Well it looks like the actual parsing happens with https://github.com/fb55/htmlparser2 under the hood. However, there doesn't seem to be any open issues there regarding the `&nbsp;` issue. Hard to tell where the issue happens. And its been about a year since @wrakky 's last commit. Maybe a sign to look elsewhere.

**Comment by rmccue** [comment]
Fixed this in #14; it just needs an extra flag passed to htmlparser2.

**Comment by wrakky** [comment]
This will be fixed in next release thanks to #14 and #17
